pyproject.toml

```diff
@@ -45,14 +45,15 @@ dependencies = [
 
 [project.optional-dependencies]
 dev = [
-  "hatch==1.9.1",
-  "ruff==0.1.6",
   "djlint==1.34.1",
-  "mkdocs==1.5.3",
+  "hatch==1.9.1",
+  "isort==5.12",
   "mkdocs-material==9.4.14",
-  "vermin==1.6.0",
+  "mkdocs==1.5.3",
   "pymdown-extensions==10.4",
-  "isort==5.12",
+  "ruff==0.1.6",
+  "sentry-sdk==1.39.2",
```

Comment:
I put it as a `dev` dependency to not force it but we might want to create a dedicated `optional-dependencies` section.

Reply:
having this as a `dev` dependency seems off to me (we shouldn't need dev tools on the server we deploy to)

Reply:
Alright, I moved it to a dedicated section 👍 

🔖  update the deployment script!